modules/dev-utils/src/main/java/org/apache/ignite/development/utils/IgniteWalConverterArguments.java

```diff
@@ -113,33 +113,33 @@
      * @param walDir                        Path to dir with wal files.
      * @param walArchiveDir                 Path to dir with archive wal files.
      * @param pageSize                      Size of pages, which was selected for file store (1024, 2048, 4096, etc).
-     * @param binaryMetadataFileStoreDir    Path to binary metadata dir.
-     * @param marshallerMappingFileStoreDir Path to marshaller dir.
+     * @param binaryMetadataDir             Path to binary metadata dir.
+     * @param marshallerMappingDir          Path to marshaller dir.
      * @param keepBinary                    Keep binary flag.
      * @param recordTypes                   WAL record types (TX_RECORD, DATA_RECORD, etc).
      * @param fromTime                      The start time interval for the record time in milliseconds.
      * @param toTime                        The end time interval for the record time in milliseconds.
-     * @param recordContainsText            Filter by substring in the WAL record.
-     * @param processSensitiveData          Strategy for the processing of sensitive data (SHOW, HIDE, HASH, MD5).
+     * @param hasText                       Filter by substring in the WAL record.
+     * @param includeSensitive              Strategy for the processing of sensitive data (SHOW, HIDE, HASH, MD5).
      * @param printStat                     Write summary statistics for WAL.
      * @param skipCrc                       Skip CRC calculation/check flag.
      */
     public IgniteWalConverterArguments(File walDir, File walArchiveDir, int pageSize,
-        File binaryMetadataFileStoreDir, File marshallerMappingFileStoreDir, boolean keepBinary,
-        Set<WALRecord.RecordType> recordTypes, Long fromTime, Long toTime, String recordContainsText,
-        ProcessSensitiveData processSensitiveData,
+        File binaryMetadataDir, File marshallerMappingDir, boolean keepBinary,
+        Set<WALRecord.RecordType> recordTypes, Long fromTime, Long toTime, String hasText,
+        ProcessSensitiveData includeSensitive,
         boolean printStat, boolean skipCrc) {
```

Comment:
In this method list of arguments and method's body aren't separated. Although it's not connected to the ticket I think it's better to move '{' to separate line since you are changing this place.